terraform/cluster/aws/network.tf

```diff
@@ -4,7 +4,8 @@ locals {
     Name = var.name
     Rack = var.name
   })
-  vpc_id = var.vpc_id == "" ? aws_vpc.nodes[0].id : var.vpc_id
+  vpc_id              = var.vpc_id == "" ? aws_vpc.nodes[0].id : var.vpc_id
+  private_subnets_ids = length(var.private_subnets_ids) != 0 ? var.private_subnets_ids : aws_subnet.private[*].id
```

Comment:
wondering what would be the value of `local.private_subnets_ids` in case of `var.private = false`. and `var.private_subnets_ids` is empty, 
null?   

also, I did not find where we are using `local.private_subnets_ids`